content/post/_index.es.md

```
@@ -0,0 +1,40 @@
+- **2011 – 2012** – Época Obama (_{{< postcount 2011 2012 >}}_)
+- **2009 – 2011** – Fase experimental 1 (_{{< postcount 2009 2011 >}}_)
+- **2005 – 2009** – Threadless (_{{< postcount 2005 2009 >}}_)
+- **2004 – 2005** – Vida nocturna y segundo trabajo (_{{< postcount 2004 2005 >}}_)
+- **2001 – 2004** – Vida nocturna y primer trabajo (_{{< postcount 2001 2004 >}}_)
+- **2000 – 2001** – Universidad (_{{< postcount 2000 2001 >}}_)
+
+</details>
+
+Total de entradas desde 2000: **{{< postcount 2000 "now" >}}**
+
+## Todas las entradas
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)